internal/m-agent/messages/messages.go

```diff
@@ -47,3 +48,25 @@ func SendMessageToClient(conn *websocket.Conn, action, reqID string, payload int
 
 	return conn.WriteJSON(Message{action, payload, reqID})
 }
+
+// HandleActionExecutionError handles an error generated while performing the action requested by the client
+func HandleActionExecutionError(conn *websocket.Conn, reqID, errorCode string, err error, logger *log.Logger) {
+
+	if err := SendMessageToClient(conn, "ERROR", reqID, errorCode+err.Error()); err != nil {
+		logger.Printf("Error occured while sending error message to client, err: %v", err)
+	}
+
+	if err := conn.Close(); err != nil {
+		logger.Printf("Error occured while closing the connection, err: %v", err)
+	}
+}
+
+// HandleFeedbackTransmissionError handles an error generated while sending the feedback for an earlier requested action to the client
+func HandleFeedbackTransmissionError(conn *websocket.Conn, err error, logger *log.Logger) {
+
+	logger.Printf("Error occured while sending feedback message to client, err: %v", err)
```

Comment:
@neelanjan00 this sounds a bit cryptic - do we familiarize users with what is meant by feedback message ?

Reply:
Currently, we don't have a design document or contribution document but we can explain the same there I guess. At the moment, the context for action/feedback is also used in the API docs too: https://litmuschaos.github.io/m-agent/process-kill#message-message